package.json

```diff
@@ -108,6 +108,7 @@
     "commander": "^2.19.0",
     "debug": "^4.0.1",
     "dotenv": "^6.0.0",
+    "emoji-regex": "^7.0.1",
```

Comment:
Gautham couldn't compile without this, any idea why @schmidsi

Reply:
Actually no. This is very strange and shouldn't happen.